Cyral/Cyral.json

```diff
@@ -0,0 +1,1889 @@
+                            "timeSource": "Message"
+                        }
+                    ],
+                    "description": "",
+                    "timeRange": null,
+                    "coloringRules": null,
+                    "linkedDashboards": []
+                },
+                {
+                    "id": null,
+                    "key": "panel9FBED8429A57DB4D",
+                    "title": "Top {{topKLimit}} Policies Violated",
+                    "visualSettings": "{\"general\":{\"mode\":\"distribution\",\"type\":\"bar\"},\"series\":{},\"overrides\":[],\"axes\":{\"axisX\":{\"title\":\"Policy\",\"hideLabels\":true},\"axisY\":{\"title\":\"Total Violations\"}}}",
+                    "keepVisualSettingsConsistentWithParent": true,
+                    "panelType": "SumoSearchPanel",
+                    "queries": [
+                        {
+                            "queryString": "($$logsrc )\n| json \"policyViolated\",\"sidecar.name\",\"repo.name\" nodrop\n| where %\"policyViolated\" = \"true\" and %\"sidecar.name\" matches \"{{sidecar_name}}\" and %\"repo.name\" matches \"{{repo_name}}\"\n| json \"policyViolations[*].policyName\" as policyName\n| count as Violations by policyName\n| sort by Violations desc\n| limit {{topKLimit}}",
+                            "queryType": "Logs",
+                            "queryKey": "A",
+                            "metricsQueryMode": null,
+                            "metricsQueryData": null,
+                            "tracesQueryData": null,
+                            "parseMode": "Manual",
+                            "timeSource": "Message"
+                        }
+                    ],
+                    "description": "",
+                    "timeRange": null,
+                    "coloringRules": null,
+                    "linkedDashboards": []
+                },
+                {
+                    "id": null,
+                    "key": "panel01F1D844AF5A4A4C",
+                    "title": "Top {{topKLimit}} Policies Not Used in Past 30 Days",
+                    "visualSettings": "{\"general\":{\"mode\":\"distribution\",\"type\":\"bar\"},\"series\":{},\"overrides\":[]}",
+                    "keepVisualSettingsConsistentWithParent": true,
+                    "panelType": "SumoSearchPanel",
+                    "queries": [
+                        {
+                            "queryString": "($$logsrc )\n| json \"policyViolated\",\"sidecar.name\",\"repo.name\",\"activityTime\" nodrop\n| json \"policyViolations[*].policyName\" as policyName nodrop\n| where %\"policyViolated\" matches \"*\" and policyName=\"*\" and %\"sidecar.name\" matches \"{{sidecar_name}}\" and %\"repo.name\" matches \"{{repo_name}}\"\n| first(%\"activityTime\") by policyName| now()-30d as expireTime|parseDate(_first, \"yyyy-MM-dd HH:mm:ss\") as parsedDate\n| where expireTime > parsedDate|count as Count by policyName\n| sort by Count desc\n| limit {{topKLimit}}",
```

Comment:
Unparsed date error here as well.
Also there is a bug instead of policyName="*" it should be policyName matches "*"

Reply:
I will fix my fabricated logs to show the correct date but I believe that these parseDate errors can be ignored  since they are erroring on my fabricated logs that have incorrect date formats.

Corrected the bug